packages/common/src/store/transactions.slice.ts

```diff
@@ -71,6 +71,20 @@ export const getApprovedTransactions = createSelector(getTxHistory, (history) =>
   history.filter((h) => h.result === TxResult.APPROVED)
 );
 
+export const getApprovedTransactionsLength = createSelector(
+  getApprovedTransactions,
+  (history) => history.length
+);
+
+export const getDeniedTransactions = createSelector(getTxHistory, (history) =>
+  history.filter((h) => h.result === TxResult.DENIED)
+);
+
+export const getDeniedTransactionsLength = createSelector(
+  getDeniedTransactions,
+  (history) => history.length
+);
```

Comment:
I'm assuming this is untested because it's in the common package?

Reply:
yep